src/coreclr/dlls/mscordbi/CMakeLists.txt

```diff
@@ -102,7 +102,17 @@ elseif(CLR_CMAKE_HOST_UNIX)
     list(APPEND COREDBI_LIBRARIES
         mdhotdata_full
         palrt
-        # share the PAL in the dac module
+    )
+
+    # Due to the pal_redefines_file adding redirects
```

Comment:
Why is this change needed?

Reply:
Because for some inscrutable reason the exports in question don't exist on Linux... Oh gosh. I just realized why... I might be able to fix this. Thank you for asking!